Applications/Games/Batman™: Arkham Origins/Steam/script.js

```diff
@@ -0,0 +1,16 @@
+include(["Engines", "Wine", "QuickScript", "SteamScript"]);
+
+new SteamScript()
+    .name("Batman™: Arkham Origins")              
+    .editor("WB Games Montreal, Splash Damage")     
+    .author("ImperatorS79")
+    .wineVersion("2.15")
+    .wineDistribution("staging")
+    .postInstall(function(wine, wizard) {
+        //Steam says only compatible with windows > xp
+        wine.setOsForApplication().set("BatmanOrigins.exe", "win7").do();
```

Comment:
win7 should be the default for this Wine version, right?

Reply:
I do not know

Reply:
I think so. Please try it. We should not specify superfluous settings.

Reply:
You were right.